src/services/api/api.service.ts

```diff
@@ -0,0 +1,23 @@
+const CREDENTIALS_ORIGIN = import.meta.env.VITE_CREDENTIALS_ORIGIN
+
+export async function fetchApi(
```

Comment:
Should we use this for all API calls?

Reply:
yes indeed, but this one need an improvement for GET requests, done in another PR: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/pull/173/files#diff-7b7c2b32fd8b93dcb739fbf7d06d4d2597048988600abb55494a10dc97d10bc7R3
maybe I should include these changes in this PR too.